src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -225,11 +237,16 @@ export const FilesAndLicensePanel = ({ uppy, canEditFiles }: FilesAndLicensePane
                                   return false;
                                 });
 
+                                const isEmbargoedDegreeFile = isProtectedDegree && isEmbargoed(file.embargoDate);
+                                const canEditThisFile = isEmbargoedDegreeFile
+                                  ? canEditDegreeFiles && !!user.isEmbargoThesisCurator
```

Comment:
Denne kan vel aldri være undefined
```suggestion
                                const canEditThisFile = isEmbargoedDegreeFile
                                  ? canEditDegreeFiles && !user.isEmbargoThesisCurator
```